docs/cel2/operators/how-it-works.md

```diff
@@ -0,0 +1,29 @@
+# How it works
+
+The first set of active RPC providers were determined in the genesis block of Celo L2. Thereafter at the end of every epoch, approximately 24 hours, an election is run that may lead to nodes being added or removed from the elected set.
```

Comment:
Maybe link to what-is-celo/using-celo/protocol/epoch-rewards for anyone wondering why an epoch is a non-fixed amount of time or blocks